Public/Src/Engine/Processes/SharedOpaqueOutputHelper.cs

```diff
@@ -180,19 +180,56 @@ private static bool IsSharedOpaqueOutputWithFallback(string expandedPath, bool c
             }
         }
 
+        private const int MaxNumberOfAttemptsForMarkingSharedOpaqueOutputs = 3;
+        private static readonly TimeSpan SleepDurationBetweenMarkingAttempts = TimeSpan.FromMilliseconds(100);

```

Comment:
Isn't 100ms a little bit too much? I'd have said 10ms to begin with

Reply:
Will change to 10s and will add a linear backoff